src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,43 @@
 'use strict';
 
 class Animal {
-  // write your code here
-}
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 
+  static delleteDeadAnimals() {
+    Animal.alive = Animal.alive.filter((animal) => animal.health > 0);
+  }
```

Comment:
The method name `delleteDeadAnimals` contains a typo. It should be `deleteDeadAnimals` to correctly reflect its functionality and follow proper English spelling.